h/presenters/group_json.py

```diff
@@ -17,21 +17,24 @@ def _model(self, group):
         model = {
           'name': group.name,
           'id': group.pubid,
-          'public': group.is_public,
+          'public': group.is_public,  # DEPRECATED: TODO: remove from client
           'scoped': True if group.scopes else False,
           'type': group.type
         }
         model = self._inject_urls(group, model)
         return model
 
     def _inject_urls(self, group, model):
-        model['urls'] = {}
+        model['links'] = {}
+        model['urls'] = {}  # DEPRECATED TODO: remove from client
         if not self._links_svc:
             return model
 
-        model['urls'] = self._links_svc.get_all(group)
-        if 'group' in model['urls']:
-            model['url'] = model['urls']['group']
+        model['links'] = self._links_svc.get_all(group)
+        model['urls'] = model['links']  # DEPRECATED TODO: remove from client
```

Comment:
Watch out: I think any modifications to `model['urls']` will also silently modify `model['links']` and vice-versa. urls is going away ASAP though so we should be able to get away with it